### ISSUE TITLE
Compilation error since v0.6.1 in the Drop implementation of TesseractText

### ISSUE THREAD

**Issue by valnoel** [antimatter15/tesseract-rs#22]
Hi,

I have this compilation issue with the 0.6.1 version :
```
   Compiling leptonica-sys v0.3.2
   Compiling tesseract-sys v0.5.3
   Compiling tesseract v0.6.1 (/home/.../tesseract-rs)
error[E0308]: mismatched types
  --> src/plumbing/tesseract_text.rs:13:33
   |
13 |         unsafe { TessDeleteText(self.0) }
   |                                 ^^^^^^ types differ in mutability
   |
   = note: expected raw pointer `*mut i8`
              found raw pointer `*const i8`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
error: could not compile `tesseract`.

To learn more, run the command again with --verbose.
```

The problem seems to come from `tesseract-sys v0.5.3` where the mutability of the expected pointer has been changed from `*mut c_char` ([here](https://docs.rs/tesseract-sys/0.5.2/tesseract_sys/fn.TessDeleteText.html)) to `*const c_char` ([here](https://docs.rs/tesseract-sys/0.5.3/tesseract_sys/fn.TessDeleteText.html)), so I don't understand why a `*mut i8` is expected here by the compiler?

Thanks for your help!

**Comment by ccouzens** [comment]
Thanks for the report.

I don't get the issue on my Laptop (Fedora 32), but am seeing it on [Github actions](https://github.com/antimatter15/tesseract-rs/pull/24/checks?check_run_id=1042243409). Thank you for suggesting I add CI, it's a good idea.

Tesseract-sys didn't [change](https://github.com/ccouzens/tesseract-sys/compare/0.5.2...0.5.3) much between those 2 versions. Possibly it came from upgrading `bindgen`. Or maybe the `tesseract/capi.h` has been updated in newer versions of Ubuntu.

I will investigate and try and find a solution.

**Comment by ccouzens** [comment]
Hi @valnoel,

I've released [0.6.2](https://crates.io/crates/tesseract/0.6.2) which I think should fix this issue.

Thanks for using Tesseract-rs, I hope you enjoy it 🙂

**Comment by valnoel** [comment]
Thanks @ccouzens, this fixes my issue! :+1:
